views/ConnectionView.qml

```diff
@@ -0,0 +1,135 @@
+import QtQuick
+import QtQuick.Layouts
+import QtQuick.Controls
+import QtQuick.Dialogs
+
+
+Item {
+  id: connectionItem
+
+  ColumnLayout {
+    id: laylayout
+    width: parent.width
+    anchors.left: parent.left
+    anchors.leftMargin: 20
+    anchors.right: parent.right
+    anchors.rightMargin: 20
+    anchors.bottom: parent.bottom
+    anchors.top: parent.top
+    TextField {
```

Comment:
Please add labels to all the text fields, it is not clear, which value is modified